src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy

```diff
@@ -15,23 +15,38 @@
  */
 package com.github.eirnym.js2p
 
-
 import org.gradle.api.GradleException
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPluginConvention
 
 /**
  * Registers the plugin's tasks.
  *
  * @author Ben Manes (ben.manes@gmail.com)
+ * @author Arseny Nasokin (eirnym@gmail.com)
```

Comment:
I would remove authors complete. When you add one ? It makes only maintenance, or a wrong history.
You see it in git history. But this is just my piont of view

Reply:
makes sense